obj/atlas/structMapAutogen.go

```diff
@@ -8,21 +8,26 @@ import (
 )
 
 func AutogenerateStructMapEntry(rt reflect.Type) *AtlasEntry {
-	return AutogenerateStructMapEntryUsingTags(rt, "refmt")
+	return AutogenerateStructMapEntryUsingTags(rt, "refmt", StructFieldSort_Default)
 }
 
-func AutogenerateStructMapEntryUsingTags(rt reflect.Type, tagName string) *AtlasEntry {
+func AutogenerateStructMapEntryUsingTags(rt reflect.Type, tagName, sorter string) *AtlasEntry {
 	entry := &AtlasEntry{
 		Type:      rt,
-		StructMap: &StructMap{Fields: exploreFields(rt, tagName)},
+		StructMap: &StructMap{Fields: exploreFields(rt, tagName, sorter)},
 	}
 	return entry
 }
 
+const (
+	StructFieldSort_Default = "default"
```

Comment:
Missing a type for the consts, plz.

Maybe this is actually the same type and enum when talking about either maps or structs?